### ISSUE TITLE
nextPage() does not forward 'appsecret_proof'

### ISSUE THREAD

**Issue by lawrencebrooks** [facebook/facebook-java-business-sdk#159]
When calling the nextPage() method on 'APINodeList', the 'appsecret_proof' parameter is not forwarded on from the original request, causing an error of:

{"error":{"message":"API calls from the server require an appsecret_proof argument","type":"GraphMethodException","code":100,"fbtrace_id":"*****"}}

**Comment by frankgrimes97** [comment]
We've also hit this bug when testing an upgrade to version 3.0.0. It seems to work fine with 2.11.1.

**Comment by frankgrimes97** [comment]
Here's a quick fix/workaround: 
```
diff --git a/src/main/java/com/facebook/ads/sdk/APINodeList.java b/src/main/java/com/facebook/ads/sdk/APINodeList.java
index 9d474cf..aab8034 100644
--- a/src/main/java/com/facebook/ads/sdk/APINodeList.java
+++ b/src/main/java/com/facebook/ads/sdk/APINodeList.java
@@ -59,10 +59,6 @@ public class APINodeList<T extends APINode> extends ArrayList<T> implements APIR
     }
 
     public APINodeList<T> nextPage(int limit) throws APIException {
-      if (this.next != null) {
-        this.request.setOverrideUrl(this.next);
-        return (APINodeList<T>) request.execute();
-      }
       if (after == null) return null;
       this.request.setOverrideUrl(null);
       Map<String, Object> extraParams = new HashMap<String, Object>();
```

**Comment by frankgrimes97** [comment]
@shootingsyh Can this bug be addressed sometime before v2.11 is EOL on August 7, 2018? https://developers.facebook.com/docs/apps/changelog
We'd really like to avoid forking the project and maintaining our own release, if possible.

**Comment by mcross1991** [comment]
Any update on this as our implementations are now breaking because of this?
